assets/sass/components/ads/_googlesitekit-ads-setup-module.scss

```diff
@@ -33,6 +33,15 @@
 			margin: 0;
 		}
 
+		.googlesitekit-text-field-conversion-tracking-id-prefix.mdc-text-field__icon {
+			top: 17.5px;
+		}
+
+		.mdc-text-field--error:not(.mdc-text-field--disabled, .mdc-text-field--focused) .mdc-floating-label {
+			font-size: $fs-body-md;
+			padding: 0.5px 0;
+		}
```

Comment:
As these styles are specific to the Conversion Tracking ID text field, do you think it'd be nicer to add these to its own specific stylesheet - `assets/sass/components/ads/_googlesitekit-ads-conversion-id.scss`?

Reply:
Yes you're right, I've moved it to the other scss file in the [new PR here](https://github.com/google/site-kit-wp/pull/8469).